validation/src/validation_service/mod.rs

```diff
@@ -334,11 +334,13 @@ impl<C, N, P, SP> ParachainValidationInstances<C, N, P, SP> where
 			}
 		}
 
+		let signing_context = self.client.runtime_api().signing_context(&id)?;
```

Comment:
This needs to be guarded. As otherwise we try to call an api that is not yet available.

Reply:
So we should just error out of this function in this case? I think that then the node will not be able to do validation work until the runtime is upgraded.

Reply:
If the api function can not be found, an error is returned, so yes^^

Reply:
Ok, I bumped `ParachainHost` and added a guard.

Reply:
@bkchr This should not be triggered during major sync, regardless.